packages/lesswrong/components/ea-forum/wrapped/EAForumWrappedPage.tsx

```diff
@@ -1524,18 +1569,19 @@ const RecommendationsSection = ({classes}: {
 /**
  * Section that displays all the user's upvoted posts and lets them mark which were "most valuable"
  */
-const MostValuablePostsSection = ({classes}: {
+const MostValuablePostsSection = ({year, classes}: {
```

Comment:
Just above here the recommendations still use '2023_wrapped' as the postId, is that still right?

Reply:
Ok reading the rest of the file I can see it's more making it work for 2022 rather than making literally everything generic, so I'm assuming this is fine

Reply:
Yeah I think that is basically just a placeholder string